ansible/roles/nexus-container/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+## Sonatype Nexus Install
+
+##
+  - name: Download Nexus template to master
+    get_url:
+      url: https://raw.githubusercontent.com/OpenShiftDemos/nexus/master/nexus{{nexus_version}}-persistent-template.yaml
+      dest: /root/nexus-template.yaml
+      mode: 0440
+      
+  - name: Check if Nexus was already provisioned
+    command: "oc get service nexus -n {{desired_project}}"
+    register: install_nexus
+    ignore_errors: true
+
+### We should add a check if a pv is already created (nexus requests a 10Gi pv)
+  - name: Instantiate Nexus from template
+    command: "oc process -f /root/nexus-template.yaml --param=VOLUME_CAPACITY=10Gi --param=MAX_MEMORY=2Gi -n {{desired_project}} | oc create -f - -n {{desired_project}}"
```

Comment:
just a comment for later: wouldn't be better to have params as variables defined in playbook vars (with defaults in `roles/nexus/defaults/` set to those values) ?

Reply:
Possibly. I keep them as is for the PR but definitely something to consider for future.